app/src/androidTest/java/com/swent/assos/NewsDetailsTest.kt

```diff
@@ -107,13 +107,11 @@ class NewsDetailsTest : SuperTest() {
     run {
       ComposeScreen.onComposeScreen<NewsDetailsScreen>(composeTestRule) {
         step("Delete News") {
-          composeTestRule.waitUntil(10000) {
-            composeTestRule.onNodeWithTag("DeleteButton").isDisplayed()
-          }
-          composeTestRule.onNodeWithTag("DeleteButton").performClick()
+          composeTestRule.waitUntil(10000) { composeTestRule.onNodeWithTag("Button").isDisplayed() }
+          composeTestRule.onNodeWithTag("Button").performClick()
```

Comment:
Why did we change the name of button for deleting ?

Reply:
Because I deleted the name of the button so the test tag is just `"" + Button`